statement.go

```diff
@@ -467,6 +505,13 @@ func (s DeleteStatement) Keys() Keys {
 	return s.keys
 }
 
+// WithClusteringSentinel allows you to specify whether the use of the
+// clustering sentinel value is enabled
```

Comment:
why not just call it "clusteringSentinelsEnabled" or something then? "allow" cluster sentinel reads to me like we are permitting some existing thing to continue, whereas we're actually introducing them

Reply:
That's a good point, I think `clusteringSentinelsEnabled` sounds better too. Changed!